src/components/package/show/components/package-settings/package-settings.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+import hasIn from 'lodash/fp/hasIn';
+
+import {
+  Accordion,
+  Headline,
+  KeyValue,
+  Icon,
+} from '@folio/stripes/components';
+
+import ProxyDisplay from '../../../../proxy-display';
+import TokenDisplay from '../../../../token-display';
+import AccessType from '../../../../access-type-display';
+
+import {
+  getAccessTypeId,
+  getAccessTypeIdsAndNames,
+} from '../../../../utilities';
+import {
+  accessTypesReduxStateShape,
+} from '../../../../../constants';
+
+const propTypes = {
+  accessStatusTypes: accessTypesReduxStateShape.isRequired,
+  isOpen: PropTypes.bool.isRequired,
+  model: PropTypes.object.isRequired,
+  onToggle: PropTypes.func.isRequired,
+  packageAllowedToAddTitles: PropTypes.bool.isRequired,
+  packageSelected: PropTypes.bool.isRequired,
+  provider: PropTypes.object.isRequired,
+  proxyTypes: PropTypes.object.isRequired,
+};
+
+const PackageSettings = ({
+  isOpen,
+  onToggle,
+  model,
+  proxyTypes,
+  provider,
+  accessStatusTypes,
+  packageAllowedToAddTitles,
+  packageSelected,
+}) => {
+  const renderAccessTypeDisplay = () => {
+    if (!accessStatusTypes?.items?.data?.length) {
+      return null;
+    }
+
+    const formattedAccessTypes = getAccessTypeIdsAndNames(accessStatusTypes.items.data);
+
+    return (
+      <AccessType
+        accessTypeId={getAccessTypeId(model)}
+        accessStatusTypes={formattedAccessTypes}
+      />
+    );
+  };
+
+  const renderPackageSettings = () => {
+    const visibilityMessage = model.visibilityData.reason && `(${model.visibilityData.reason})`;
+    const hasProxy = hasIn('proxy.id', model);
+    const hasProviderToken = hasIn('providerToken.prompt', provider);
+    const hasPackageToken = hasIn('packageToken.prompt', model);
+    const isProxyAvailable = hasProxy && proxyTypes.request.isResolved && model.isLoaded && provider.isLoaded;
+    const haveAccessTypesLoaded = !accessStatusTypes?.isLoading && !model.isLoading;
+
+    return (
+      <div>
```

Comment:
Please, check all `div`s  and delete them if they haven't any data attribute or does not change layout